cycledash/static/css/runs.css

```diff
@@ -43,6 +43,10 @@ td span.run-id {
   font-size: 0.92em;
 }
 
+.runs .run-id, .runs .date {
```

Comment:
I prefer it with `.run-id`, `.date` and friends. That way the display stuff stays entirely in the CSS file and the classes in the HTML file have clear non-visual meanings.